src/test/java/com/sugarcrm/candybean/runner/BlockListUnitTest.java

```diff
@@ -30,6 +30,7 @@
 import com.sugarcrm.candybean.runner.VRunner;
 
 @RunWith(VRunner.class)
+@Ignore
```

Comment:
We should specify why this test is blocked as a string argument, so we know under what conditions it should be unblocked.  Also, if a related ticket exists (which it probably should), it should be cited in the reason.  Here's a made-up example:

@Ignore("CB-276: Running this test class causes your hard drive to be erased, your computer to catch on fire, and your landlord to evict you.  Since the code under test is never used anyway, we can clean this up later.")